languages/python/exercises/concept/strings/example.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+LOGLINE_RE = r"\[(\w*)\] (.*)"
```

Comment:
Seems odd to me that we mention in the .meta/design.md that regex should _not_ be taught here, but that's the example we choose?

Reply:
Also, this pattern is problematically loose, since it will match WAY more than INFO, WARNING, and ERROR.

Reply:
Good point. It is a cleaner solution, but not within the parameters of this concept exercise. I updated with a third option.